webrender/src/renderer.rs

```diff
@@ -552,6 +553,11 @@ impl Renderer {
         let data64_texture = VertexDataTexture::new(&mut device);
         let data128_texture = VertexDataTexture::new(&mut device);
 
+        let dummy_mask_texture = device.create_texture_ids(1)[0];
```

Comment:
I had a hard time figuring out how to properly assign a default masking texture for this case. Ended up introducing `DummyResources` struct to store the white texture as well the opaque one (which were previously forgotten but still created). Let me know if there is a better solution please.